### ISSUE TITLE
Ubuntu Apache + PHP Always shows "No way"

### ISSUE THREAD

**Issue by xanderdunn** [simogeo/Filemanager#414]
I'm on Ubuntu 14.04 with Apache 2.4.7.  I have a copy of Filemanager at /sw/Filemanager.  

In my apache config file for the host, I have `DocumentRoot /sw/`, so it should be located correctly.  

My /sw/Filemanager/scripts/filemanager.config.js:
```
{
		"_comment": "IMPORTANT : go to the wiki page to know about options configuration https://github.com/simogeo/Filemanager/wiki/Filemanager-configuration-file",
    "options": {
        "culture": "en",
        "lang": "php",
        "theme": "flat-dark",
        "defaultViewMode": "grid",
        "autoload": true,
        "showFullPath": false,
        "showTitleAttr": false,
        "browseOnly": false,
        "showConfirmation": true,
        "showThumbs": true,
        "generateThumbnails": true,
        "searchBox": true,
        "listFiles": true,
        "fileSorting": "default",
        "chars_only_latin": true,
        "dateFormat": "d M Y H:i",
        "serverRoot": true,
        "fileRoot": false,
        "baseUrl": false,
        "logger": true,
        "capabilities": ["select", "download", "rename", "delete", "replace"],
        "plugins": []
    },
    "security": {
        "allowFolderDownload": false,
        "allowChangeExtensions": false,
        "allowNoExtension": false,
        "uploadPolicy": "DISALLOW_ALL",
        "uploadRestrictions": [
            "jpg",
            "jpe",
            "jpeg",
            "gif",
            "png",
            "svg",
            "txt",
            "pdf",
            "odp",
            "ods",
            "odt",
            "rtf",
            "doc",
            "docx",
            "xls",
            "xlsx",
            "ppt",
            "pptx",
            "csv",
            "ogv",
            "mp4",
            "webm",
            "m4v",
            "ogg",
            "mp3",
            "wav",
            "zip",
            "rar"
        ]
    },
    "upload": {
        "multiple": true,
        "number": 5,
        "overwrite": false,
        "imagesOnly": false,
        "fileSizeLimit": 16
    },
    "exclude": {
        "unallowed_files": [
            ".htaccess",
            "web.config"
        ],
        "unallowed_dirs": [
            "_thumbs",
            ".CDN_ACCESS_LOGS",
            "cloudservers"
        ],
        "unallowed_files_REGEXP": "/^\\./",
        "unallowed_dirs_REGEXP": "/^\\./"
    },
    "images": {
        "imagesExt": [
            "jpg",
            "jpe",
            "jpeg",
            "gif",
            "png",
            "svg"
        ],
        "resize": {
        	"enabled":true,
        	"maxWidth": 1280,
            "maxHeight": 1024
        }
    },
    "videos": {
        "showVideoPlayer": true,
        "videosExt": [
            "ogv",
            "mp4",
            "webm",
            "m4v"
        ],
        "videosPlayerWidth": 400,
        "videosPlayerHeight": 222
    },
    "audios": {
        "showAudioPlayer": true,
        "audiosExt": [
            "ogg",
            "mp3",
            "wav"
        ]
    },
    "pdfs": {
        "showPdfReader": true,
        "pdfsExt": [
            "pdf",
            "odp"
        ],
	    "pdfsReaderWidth": "640",
        "pdfsReaderHeight": "480"
    },
    "edit": {
        "enabled": true,
        "lineNumbers": true,
        "lineWrapping": true,
        "codeHighlight": false,
        "theme": "elegant",
        "editExt": [
            "txt",
            "csv"
        ]
    },
    "customScrollbar": {
    	"enabled": true,
    	"theme": "inset-2-dark",
    	"button": true
    },
    "extras": {
        "extra_js": [],
        "extra_js_async": true
    },
    "icons": {
        "path": "images/fileicons/",
        "directory": "_Open.png",
        "default": "default.png"
    },
    "url": "https://github.com/simogeo/Filemanager",
    "version": "2.0.0-dev"
}
```

When I load the page, I see this in /var/log/apache2/error.log:
```
PHP Warning:  file_get_contents(../../scripts/filemanager.config.js.default): failed to open stream: No such file or directory in /sw/Filemanager/connectors/php/filemanager.class.php on line 38, referer: http://mypage.net/Filemanager/
PHP Warning:  array_replace_recursive(): Argument #1 is not an array in /sw/Filemanager/connectors/php/filemanager.class.php on line 48, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: logger in /sw/Filemanager/connectors/php/filemanager.class.php on line 69, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 79, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: serverRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 80, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 85, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 86, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 87, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: capabilities in /sw/Filemanager/connectors/php/filemanager.class.php on line 967, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: edit in /sw/Filemanager/connectors/php/filemanager.class.php on line 969, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: culture in /sw/Filemanager/connectors/php/filemanager.class.php on line 1380, referer: http://mypage.net/Filemanager/
PHP Warning:  file_get_contents(/sw/Filemanager/scripts/languages/.js): failed to open stream: No such file or directory in /sw/Filemanager/connectors/php/filemanager.class.php on line 1387, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 1063, referer: http://mypage.net/Filemanager/
PHP Warning:  file_get_contents(../../scripts/filemanager.config.js.default): failed to open stream: No such file or directory in /sw/Filemanager/connectors/php/filemanager.class.php on line 38, referer: http://mypage.net/Filemanager/
PHP Warning:  array_replace_recursive(): Argument #1 is not an array in /sw/Filemanager/connectors/php/filemanager.class.php on line 48, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: logger in /sw/Filemanager/connectors/php/filemanager.class.php on line 69, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 79, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: serverRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 80, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 85, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 86, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 87, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: capabilities in /sw/Filemanager/connectors/php/filemanager.class.php on line 967, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: edit in /sw/Filemanager/connectors/php/filemanager.class.php on line 969, referer: http://mypage.net/Filemanager/
[Thu Nov 05 16:41:24.421700 2015] [:error] [pid 21008] [client 10.24.0.109:58558] PHP Notice:  Undefined index: culture in /sw/Filemanager/connectors/php/filemanager.class.php on line 1380, referer: http://mypage.net/Filemanager/
PHP Warning:  file_get_contents(/sw/Filemanager/scripts/languages/.js): failed to open stream: No such file or directory in /sw/Filemanager/connectors/php/filemanager.class.php on line 1387, referer: http://mypage.net/Filemanager/
PHP Notice:  Undefined index: fileRoot in /sw/Filemanager/connectors/php/filemanager.class.php on line 1063, referer: http://mypage.net/Filemanager/
```

In my /etc/apache2/apache2.conf I have:
```
<Directory />
        Options Indexes FollowSymLinks Includes ExecCGI
        AllowOverride All
        Require all granted
</Directory>
```
So, it should also be allowed to do whatever it wants. 

Opening my browser to mypage.net/Filemanager always shows this:
![screen shot 2015-11-05 at 16 48 22](https://cloud.githubusercontent.com/assets/1313618/10986293/aa231250-83dd-11e5-87fd-e074ba33ac45.png)

I tried setting `"logger": true`, but nothing ever appears in /tmp/filemanager.log and I also tried `"logfile": "/var/www/html/filemanager.log"` but nothing ever appears there, even if I create an empty log file in that location.

**Comment by xanderdunn** [comment]
My problem was two-fold: 
- Rather than copying the filemanager.config.js.default, I actually moved it.  Apparently, .  Setting back into place the filemanager.config.js.default file fixed the "No way." problem.  However, still no files showed up.  
- Filemanager didn't have permission to read any of the files in the serverRoot.  So, I simply turned off serverRoot and pointed fileRoot to a directory with files that are readable by everyone.
